README.md

```diff
@@ -2,12 +2,36 @@ Build Status: [![Build Status](https://azure-functions.visualstudio.com/azure-fu
 
 CoreTools Healthcheck: [![CoreTools healthcheck](https://azure-functions.visualstudio.com/azure-functions-docker/_apis/build/status/v3%20core-tools%20image%20health%20check?branchName=dev)](https://azure-functions.visualstudio.com/azure-functions-docker/_build/latest?definitionId=43&branchName=dev)
 
+# Host Release Information
+
+Azure-Functions-Docker/host Images : 
+
+Azure-Functions-Docker releases Linux Container Images for Azure-Functions-Host.  These images consist of the[ Azure-Functions-Host](https://github.com/Azure/azure-functions-host), [Extension Bundles](https://github.com/Azure/azure-functions-extension-bundles), and a language worker (ex. [Powershell-Worker](https://github.com/Azure/azure-functions-powershell-worker)).  
+
+Host images have an expected release cadence of once a month. These images are available at the Microsoft Container Registry links visible below.  Versions 3 and 4 are currently being updated in parallel.  Major Version images, those beginning in :3.0 or :4, are the preferred images and will always host the latest released versions after thorough testing and verification. Customers that need a custom image should follow [this documentation for building a custom image](https://docs.microsoft.com/en-us/azure/azure-functions/functions-create-function-linux-custom-image). 
+
+There are a variety of Host images that are released. All images contain everything necessary to run Azure-Functions-Host
```

Comment:
Rephrase - This repo contains various flavors for dockerfiles .

Insert table here with columns Tag , Description -> add row for 3.0*tags and another row for 4* tag